portfolio/src/main/webapp/photos.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="navbar">
+        <ul>
+          <li><a href="index.html">home</a></li> 
+          <li><a href="projects.html">projects</a></li> 
+          <li><a href="blog.html">blog</a></li> 
+          <li><a href="photos.html">photos</a></li> 
+          <li><a href="random.html">random</a></li>
+        </ul>
+    </div> 
+    
+    <div id="photoscontent">
+        <div id="photosdescrip">
+            <h1>Photos + nice memories</h1>
+            <h4>Amateur photography shot on (arguably outdated) iPhone 6</h4> 
+        </div>
+        <div id="photosblocks">
+            <div class="photoblock" id="pr">
+                <img src="./images/gallery/pr.jpg">
+                <div class="phototext">
+                    <h3>Door, Old San Juan, Puerto Rico</h3>
+                    <h4>January 28, 2020</h4> 
+                </div>
+                <p>Trip with friends to San Juan, Puerto Rico over intersession (break between fall and spring semesters). 
+                    The door is painted black as a representation of resistance from the Puerto Rican people.</p>
+            </div>
+            <div class="photoblock" id="yosemite">
+                <img src="./images/gallery/halfdome.jpg">
+                <div class="phototext">
+                    <h3>Half Dome, Yosemite Valley, California</h3>
+                    <h4>August 17, 2019</h4> 
+                </div>
+                <p>Sunset on Half Dome at Glacier Point, 2-day weekend trip to Yosemite National Park. 
+                    Absolutely beautiful scenery and clear skies at night to see the Milky Way.</p>
+            </div>
+            <div class="photoblock" id="zjj">
+                <img src="./images/gallery/zjj.jpg">
```

Comment:
v nit: prefer consistency on how you handle paths: absolute path, relative paths, using ./images vs /images

i think in index.html you're just using /images/ for some of the social media links?